config/locales/two_factor_authentication/fr.yml

```diff
@@ -50,11 +50,14 @@ fr:
     max_personal_key_login_attempts_reached: Pour votre sécurité, votre compte est
       temporairement verrouillé, car vous avez entré le code de sécurité à utilisation
       unique de façon erronée à de trop nombreuses reprises.
-    max_piv_cac_login_attempts_reached: NOT TRANSLATED YET
+    max_piv_cac_login_attempts_reached: Pour votre sécurité, votre compte a été temporairement
+      bloqué en raison de la saisie de mauvais identifiants PIV/CAC à de trop nombreuses
+      reprises.
     otp_delivery_preference:
       instruction: Vous pouvez changer cette sélection la prochaine fois que vous
         vous connectez.
-      phone_unsupported: NOT TRANSLATED YET
+      phone_unsupported: Nous ne sommes pas en mesure de passer des appels téléphoniques
+        aux personnes situées à %{location} pour le moment.
```

Comment:
This one is tricky, because in French, the preposition used to refer to a country depends on the country, but it's probably not worth trying to be clever here.

Reply:
Then again, it could be an opportunity to say we care about the little things. What I'm thinking is adding a `french_preposition` key in `country_dialing_codes.yml` and conditionally pass it into the locale.

Reply:
Is this something we could create a new ticket for and handle in a followup PR?

Reply:
Yeah, for sure. I wasn't thinking we would do it in this PR.